src/api/user/theme.ts

```diff
@@ -1,5 +1,7 @@
 import apiUser from "@/api/common/apiUser";
-import type { Themetype } from "@/types/DTO/themeDTO";
+import type { Themetype, ThemeInfoResponseDTO } from "@/types/DTO/themeDTO";
+import type { cardItemData } from "@/types/DTO/productDTO";
+import type { ThemeProductsResponseDTO } from "@/types/DTO/themeDTO";
```

Comment:
포맷터 단에서 아래와 같이 중복 경로 없도록 잡아주게 처리해줘도 괜찮겠네요!

```suggestion
import type { Themetype, ThemeInfoResponseDTO, ThemeProductsResponseDTO } from "@/types/DTO/themeDTO";
import type { cardItemData } from "@/types/DTO/productDTO";
```